dronekit/__init__.py

```diff
@@ -1392,6 +1469,20 @@ def velocity(self):
         return [self._vx, self._vy, self._vz]
 
     @property
+    def version(self):
+        """
+        The autopilot version in a Version object.
+        """
+        return Version(self._raw_version, self._autopilot_type, self._vehicle_type)
+
+    @property
+    def capabilities(self):
+        """
+        The capabilities of the autopilot in a Capabilities object.
```

Comment:
Replace `Capabilities` in this sentence with
```
:py:class:`Capabilities`
```
To create a link to your Capabilities class